### ISSUE TITLE
Add footer with the details

### ISSUE THREAD

**Issue by Harshita248** [Harshita248/Scrum-Management#12]
For now add the design , styling and animation effects of a footer. Details when provided can be added. : )

**Comment by ghost** [comment]
Hello  sir can you assign this work to me

**Comment by ArjitSahu** [comment]
I want to contribute this issue

**Comment by itsmesohit** [comment]
I can create a very good looking footer , I want to work on this

**Comment by Harshita248** [comment]
Yes @Deep2631 go ahead and start working on this. Send a pr after 5th Jan 2021 . As the issue is assigned on first come first serve basis.

**Comment by imHarry404** [comment]
I want to make the footer. guide me so that I can contribute as much as possible.

**Comment by ghost** [comment]
ok thank you mam

**Comment by ghost** [comment]
for assigning me

**Comment by imHarry404** [comment]
@Harshita248 
I want to contribute to this issue

**Comment by himanshu007-creator** [comment]
Assign me the issue if not resolved till 5th january

**Comment by priyanshi-2508** [comment]
Hello, I want to work on this issue under SWOC.

**Comment by Harshita248** [comment]
Please see to it that the issue has been assigned to someone so if he takes much longer I will assign the issue to someone else.

**Comment by dollyjain51099** [comment]
I would like to work on this issue, please assign it to me?

**Comment by ghost** [comment]
I have started working on this i will send pr soon.

**Comment by priyanshi-2508** [comment]
@Harshita248 Ma'am if there's no one, please assign me that issue.

**Comment by Harshita248** [comment]
@Deep2631  is already assigned with this issue. If he can't complete we will assign it to you

**Comment by imsachinsingh10** [comment]
I have already made the footer for this website, so if you want then assign it to me or i should wait till last
